dashboards-observability/.cypress/integration/3_panels.spec.js

```diff
@@ -543,3 +596,32 @@ describe('Clean up all test data', () => {
   });
 });
 
+describe('Verify Operational Panels Table', () =>{
+  it('Add Dummy Data to Operational Panel and verify Table Column, Pagination and Rows Data ', () => {
+  var i = 0;
+  for (i = 0; i < 12 ; i++) { 
+  createDummyPanel(); }
+  cy.get('.euiTableCellContent__text').contains('Name').should('exist');
+  cy.get('.euiTableCellContent__text').contains('Last updated').should('exist');
+  cy.get('.euiTableCellContent__text').contains('Created').should('exist');
+  moveToPanelHome();
+  cy.get('[data-test-subj="pagination-button-next"]').click();
+  cy.get('.euiTableCellContent--truncateText').contains('TestPanel1').should('exist');
+  cy.get('[data-test-subj="pagination-button-previous"]').click();
+  cy.wait(delay);
+  cy.get('.euiTableCellContent--truncateText').contains('TestPanel12').should('exist');
+  cy.get('.euiButtonEmpty__text').contains('Rows per page').click();
+  cy.get('.euiContextMenuItem__text').contains('8 rows').click();
+  let expected_row_count=8;
+  cy.get('.euiTable--auto')
+  .find("tr")
+  .then((row) => {
+    let total=row.length-1;
+    expect(total).to.equal(expected_row_count);
+  })
+  var y=0;
+  for (y=0;y<2;y++){
```

Comment:
`for(let y=0; y<2;y++)` should work as well, instead of declaring y above. Any reason for choosing `y` as the variable name? I see you have used `i` for loops in the function before. This could have been `j` or `x`. Usually we use `y` after using `x` in a loop. This can make someone confused.

Reply:
chnages the variable name as per sequence like j and k